airflow/jobs/backfill_job.py

```diff
@@ -49,7 +51,7 @@ class BackfillJob(BaseJob):
     triggers a set of task instance runs, in the right order and lasts for
     as long as it takes for the set of task instance to be completed.
     """
-    ID_PREFIX = 'backfill_'
+    ID_PREFIX = DagRunType.BACKFILL_JOB
     ID_FORMAT_PREFIX = ID_PREFIX + '{0}'
```

Comment:
Do we need these constants?  Maybe it would be better to use the DagRunType directly?

Reply:
I am happy to do this in other PR.

Reply:
I will leave it for other PR. It's not strictly related to this PR.